src/lit-html.ts

```diff
@@ -346,7 +347,7 @@ export class NodePart implements SinglePart {
       // primitive?
       node.textContent = value;
     } else {
-      this._setNode(new Text(value));
+      this._setNode(document.createTextNode(value));
```

Comment:
`this._setNode(document.createTextNode(value === undefined ? '' : value));` will fix the two failing tests.